src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/DeploymentTest.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using System.Xml;
+using System.Xml.Linq;
+using Microsoft.Build.Framework;
+using Mono.Cecil;
+using NUnit.Framework;
+using Xamarin.Android.Tools;
+using Xamarin.ProjectTools;
+
+namespace Xamarin.Android.Build.Tests
+{
+
+	public class DeploymentTestTest : BaseTest {
+
+		static ProjectBuilder builder;
+		static XamarinFormsAndroidApplicationProject proj;
+
+		[OneTimeSetUp]
+		public void BeforeDeplpoymentTests ()
+		{
+			proj = new XamarinFormsAndroidApplicationProject ();
+			proj.SetProperty (KnownProperties.AndroidSupportedAbis, "armeabi-v7a;x86");
+			var mainPage = proj.Sources.First (x => x.Include () == "MainPage.xaml.cs");
+			var source = mainPage.TextContent ().Replace ("InitializeComponent ();", @"InitializeComponent ();
+			Console.WriteLine ($""TimeZoneInfo={TimeZoneInfo.Local.DisplayName}"");
+");
+			mainPage.TextContent = () => source;
+			builder = CreateApkBuilder (Path.Combine ("temp", "DeploymentTests"));
+			string apiLevel;
+			proj.TargetFrameworkVersion = builder.LatestTargetFrameworkVersion (out apiLevel);
+			proj.AndroidManifest = @"<?xml version=""1.0"" encoding=""utf-8""?>
+<manifest xmlns:android=""http://schemas.android.com/apk/res/android"" android:versionCode=""1"" android:versionName=""1.0"" package=""UnnamedProject.UnnamedProject"">
+	<uses-sdk android:minSdkVersion=""24"" android:targetSdkVersion=""{APILEVEL}"" />
+	<application android:label=""${PROJECT_NAME}"">
+	</application >
+</manifest> ".Replace ("{APILEVEL}", apiLevel);
```

Comment:
Instead of using `string.Replace()`, this can use multi-line forgettable strings:

```csharp
proj.AndroidManifest = $@"<?xml version=""1.0"" encoding=""utf-8""?>
<manifest xmlns:android=""http://schemas.android.com/apk/res/android"" android:versionCode=""1"" android:versionName=""1.0"" package=""UnnamedProject.UnnamedProject"">
	<uses-sdk android:minSdkVersion=""24"" android:targetSdkVersion=""{apiLevel}"" />
	<application android:label=""${{PROJECT_NAME}}"">
	</application >
</manifest>";
```